index.html

```diff
@@ -420,19 +420,29 @@
                 </div>
             </div>
             <!-- Nitish's btn end -->
-						
-						
-             <!-- cyphers 2nd button start -->
-             <div class="button-container">
+
+
+            <!-- cyphers 2nd button start -->
+            <div class="button-container">
                 <button class="cypher-2-btn"> Hover me </button>
                 <div class="createdby-section">
                     Created by
                     <a href="https://github.com/cypher1002">cypher1002</a>
                 </div>
             </div>
             <!-- cyphers 2nd button end -->
-          </div>
-           
+
+            <!-- gaurav-chatribin button start -->
+            <div class="button-container">
+                <button class="gaurav-hover bn26">Hover Me</button>
+                <div class="createdby-section">
+                    Created by
+                    <a href="https://github.com/gaurav-chatribin">gaurav-chatribin</a>
+                </div>
+            </div>
+            <!-- gaurav-chatribin button end -->
+        </div>
+
 
 
         </div>
```

Comment:
here u have added one more closing div and as per our 
contribution guideline we have mention how to give css class names 
see how others have done 
![image](https://user-images.githubusercontent.com/83064473/194761219-d71a4822-0cbc-47e6-8373-e1eb25dc251a.png)
and whats bn26 ( car modal no? )

and 2 classes for one button tag and also

Reply:
does this css even matters 
![image](https://user-images.githubusercontent.com/83064473/194761312-6434c100-fe91-47d6-ad61-9dcbe2154221.png)
the way it is written means both css classes are same